lib/rubocop/path_util.rb

```diff
@@ -1,17 +1,29 @@
 # frozen_string_literal: true
 
+require 'rbconfig'
+
 module RuboCop
   # Common methods and behaviors for dealing with paths.
   module PathUtil
     module_function
 
+    def windows?
+      (RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/)
```

Comment:
@jonas054 just added `Platform.windows?` in #3971 (with yet another implementation). I suppose you should use that instead.